extensions/registration-service/src/main/java/org/eclipse/dataspaceconnector/registration/AuthorityExtension.java

```diff
@@ -129,8 +130,8 @@ public ParticipantStore participantStore() {
     }
 
     @Provider(isDefault = true)
-    public CredentialsVerifier credentialsVerifier() {
-        return new DummyCredentialsVerifier();
+    public ParticipantVerifier participantVerifier(ServiceExtensionContext context) {
+        return new DefaultParticipantVerifier(context.getMonitor(), context.getService(PolicyEngine.class), context.getService(DataspaceRegistrationPolicy.class));
```

Comment:
Avoiding injection of PolicyEngine and DataspaceRegistrationPolicy because of DI issues?

Reply:
same old, same old